src/error.rs

```diff
@@ -496,6 +507,11 @@ impl Error {
                 all(any(feature = "client", feature = "server"), feature = "http1"),
                 all(feature = "server", feature = "http2")
             ))]
+            #[cfg(all(
+                any(feature = "client", feature = "server"),
+                feature = "http2"
+            ))]
+            Kind::User(User::InvalidConnectWithBody) => " user sent connect request with non-zero body via HTTP/2",
```

Comment:
```suggestion
            Kind::User(User::InvalidConnectWithBody) => "user sent CONNECT request with non-zero body",
```

How about adjusting the message a little like this?

Reply:
yeah for sure! just fixed :)